app/grandchallenge/evaluation/migrations/0070_phase_algorithm_maximum_settable_memory_gb_and_more.py

```diff
@@ -27,7 +27,7 @@ class Migration(migrations.Migration):
             model_name="phase",
             name="algorithm_selectable_gpu_type_choices",
             field=models.JSONField(
-                default=grandchallenge.evaluation.models.get_default_gpu_type_choices,
+                default=grandchallenge.components.utils.get_default_gpu_type_choices,
```

Comment:
Same as comment for migration 0069. 
These two are the only old migrations where this function is referenced.